lib/mastodon/version.rb

```diff
@@ -17,11 +17,11 @@ def patch
     end
 
     def flags
-      ENV.fetch('MASTODON_VERSION_FLAGS', '-beta2')
+      ENV.fetch('MASTODON_VERSION_FLAGS', nil).presence || '-beta2'
```

Comment:
I feel like this is more idiomatic and easier to read, but not a big deal:
```suggestion
      ENV['MASTODON_VERSION_FLAGS'].presence || '-beta2'
```